app/proc/telegram_test.go

```diff
@@ -57,6 +57,28 @@ func TestSendIfChannelIDEmpty(t *testing.T) {
 	assert.NoError(t, err)
 }
 
+func TestSendIfContentLengthZero(t *testing.T) {
+	client := TelegramClient{
+		Bot: &tb.Bot{},
+	}
+
+	ts := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		w.WriteHeader(http.StatusInternalServerError)
+		w.Header().Set("Content-Length", string(0))
+	}))
+	defer ts.Close()
+
+	err := client.Send("100500", feed.Item{
+		Enclosure: feed.Enclosure{
+			URL:    ts.URL,
+			Length: 0,
+		},
+	})
+
+	assert.NotNil(t, err)
```

Comment:
assert.Error is better here because it shows more appropriate error message on test failure.

Reply:
Done